test/e2e/adminapi_cluster_get.go

```diff
@@ -27,5 +27,6 @@ var _ = Describe("[Admin API] Get cluster action", func() {
 		// Note: some fields will have empty values
 		// on successfully provisioned cluster (oc.Properties.Install, for example)
 		Expect(oc.Properties.StorageSuffix).ToNot(BeEmpty())
+		Expect(oc.Properties.InfraID).ToNot(BeEmpty())
```

Comment:
I think it should not be empty: all new clusters should have it. Also once we run and upgrade with https://github.com/Azure/ARO-RP/pull/1536 (yet to happen) the old clusters will have it too.

Reply:
Disregard this. I misread the test. It says “To not be empty”. I read it as “to be empty”.